src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -857,15 +857,22 @@ private boolean canMineBlock(BlockPos targetPos, IBlockState existingState) {
     }
 
     protected float getLayerHardness(BlockPos targetPos, BlockRailBase.EnumRailDirection dir) {
-        float hardness = layerAction(targetPos, dir, 0F, this::getBlockHardness, (s, r) -> s + r);
+        float hardness = layerAction(targetPos, dir, 0F, (pos, direction) -> {
+            float f = getBlockHardness(pos, direction);
+            ItemStack head = getStackInSlot(0);
+            if (canHeadHarvestBlock(head, worldObj.getBlockState(pos))) {
+                int i = EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, head);
+                f /= i * i + 1;
```

Comment:
This one?
```
hardness /= (1 + EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, boreSlot));
```
No, it has very non-linear characteristics, making Level 5 nearly worthless compared to Level 1.